FF119 Add PublicKeyCredential.parseCreationOptionsFromJSON() (#29436)
mdn/content: files/en-us/web/api/credentialscontainer/get/index.md

  - : An array of objects defining a restricted list of the acceptable credentials for retrieval. Each object will contain the following properties:

    - `id`

      - : An {{jsxref("ArrayBuffer")}}, {{jsxref("TypedArray")}}, or {{jsxref("DataView")}} representing the ID of the public key credential to retrieve. This value is mirrored by the {{domxref("PublicKeyCredential.rawId", "rawId")}} property of the {{domxref("PublicKeyCredential")}} object returned by a successful `get()` call.

    - `transports`

>       - : An array of strings providing hints as to the methods the client could use to communicate with the relevant authenticator of the public key credential to retrieve. Possible transports are: `"ble"`, `"hybrid"`, `"internal"`, `"nfc"`, and `"usb"` (see {{domxref("AuthenticatorAttestationResponse.getTransports", "getTransports()")}} for more details).

Author: Probably not. All I did here was push this into a definition list - didn't look too hard at the content.
```suggestion
      - : An array of strings providing hints as to the methods the client could use to communicate with the relevant authenticator of the public key credential to retrieve. Possible transports are: `"ble"`, `"hybrid"`, `"internal"`, `"nfc"`, and `"usb"`.
```

    The value can be one of the following:

    - `"required"`
      - : The relying party requires user verification, and the operation will fail if it does not occur.
    - `"preferred"`
      - : The relying party prefers user verification if possible, but the operation will not fail if it does not occur.
    - `"discouraged"`
      - : The relying party does not want user verification, in the interests of making user interaction as smooth as possible.

    If `userVerification` is omitted, it will default to `"preferred"`.

- `hints` {{optional_inline}}

>   - : A array of "hint" strings that to guide the user-agent in what sort of UI it might use to interact with the user.

Author: ```suggestion
  - : An array of strings providing hints as to what authentication UI the user-agent should provide for the user.
```

>     The value can be one of the following:

Author: ```suggestion
    The values can be any of the following:
```
